cammac60/MacRae-Photo
# Comp setup

### What does this PR do? 

- Adds SASS vars/imports the file to all style sheets. 
- Adds default styling to App.
- Adds default state to App.
- Sets up components w/ imports and links.
- Imports all components in App.
- Adds snapshot testing and test setup to all components.

----------

### How to manually test: 

- `git fetch`
- `git checkout comp-setup`
- `npm test` to ensure all tests are passing (with the exception of `apiCalls`, which will be added later)
- 'npm start` to ensure that the app starts on localhost

----------

### Tickets: 

- Resolves #8 
- Resolves #19
